pymatviz/ptable.py

```diff
@@ -1001,6 +797,7 @@ def ptable_heatmap_splits(
     colormap: str | Colormap = "viridis",
     on_empty: Literal["hide", "show"] = "hide",
     hide_f_block: bool | None = None,
+    # TODO: I think we agreed to remove the "Callable" type of "plot_kwargs"?
     plot_kwargs: dict[str, Any]
     | Callable[[Sequence[float]], dict[str, Any]]
```

Comment:
I think we decided to remove `Callable` from here? Need to double check.

Reply:
yes, i also remember discussing that

Reply:
Thanks for confirming. I would remove this :)